From 7a4207f4ff542bf453107862473d3d6a4ddd9587 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Wed, 30 Nov 2016 11:43:26 +0800
Subject: [PATCH] arm64: dts: rk3399: add a opp-suspend property for cpu opp
 table

The opp who contains a opp-suspend property will be configured
during suspend or reboot.

Change-Id: I6b2eede43216435f568db6959127a6e84c8cd4c8
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-opp.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-opp.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-opp.dtsi
index 2d3ca3766bf9..5c38006773c0 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-opp.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-opp.dtsi
@@ -61,6 +61,7 @@
 			opp-hz = /bits/ 64 <816000000>;
 			opp-microvolt = <800000>;
 			clock-latency-ns = <40000>;
+			opp-suspend;
 		};
 		opp03 {
 			opp-hz = /bits/ 64 <1008000000>;
@@ -102,6 +103,7 @@
 			opp-hz = /bits/ 64 <816000000>;
 			opp-microvolt = <825000>;
 			clock-latency-ns = <40000>;
+			opp-suspend;
 		};
 		opp03 {
 			opp-hz = /bits/ 64 <1008000000>;
-- 
2.35.3

